src/panels/RetinaCapturePanel.ts

```diff
@@ -0,0 +1,169 @@
+        const localCaptureUri = await window.showSaveDialog({
+            defaultUri: Uri.file(this.captureFolderName),
+            saveLabel: "Download",
+            title: "Download Retina File",
+        });
+
+        if (!localCaptureUri) {
+            return;
+        }
+
+        const localCpPath = getLocalKubectlCpPath(localCaptureUri);
+
+        const createPodYaml = `
+apiVersion: v1
+kind: Pod
+metadata:
+  name: node-explorer-${node}
+spec:
+  nodeName: ${node}
+  tolerations:
+    - key: CriticalAddonsOnly
+      operator: Exists
+    - effect: NoExecute
+      operator: Exists
+    - effect: NoSchedule
+      operator: Exists
+  volumes:
+  - name: mnt-captures
+    hostPath:
+      path: /mnt/capture
+  containers:
+  - name: node-explorer
+    image: alpine
+    command: ["sleep", "9999999999"]
+    volumeMounts:
+    - name: mnt-captures
+      mountPath: /mnt/capture
+`  ;
+
+        const applyResult = await longRunning(`Deploying pod to capture ${node} retina data.`, async () => {
+            return await withOptionalTempFile(createPodYaml, "YAML", async (podSpecFile) => {
+                const command = `apply -f ${podSpecFile}`;
+                return await invokeKubectlCommand(this.kubectl, this.kubeConfigFilePath, command);
+            });
+        });
+
+        if (failed(applyResult)) {
+            vscode.window.showErrorMessage(`Failed to apply Pod: ${applyResult.error}`);
+            throw new Error(`Failed to apply Pod: ${applyResult.error}`);
+        }
+        const waitResult = await longRunning(`waiting for pod to get ready node-explorer-${node}.`, async () => {
+            const command = `wait pod -n default --for=condition=ready --timeout=300s node-explorer-${node}`;
+            return await invokeKubectlCommand(this.kubectl, this.kubeConfigFilePath, command);
+        });
+
+        if (failed(waitResult)) {
+            vscode.window.showErrorMessage(`Failed to wait for Pod to be ready: ${waitResult.error}`);
+            return;
+        }
+
+
+        // // kubectl cp 
+        const nodeExplorerResult = await longRunning(`Copy captured data to local host location ${localCpPath}.`, async () => {
+            const cpcommand = `cp node-explorer-${node}:mnt/capture ${localCpPath} --retries 99`;
+            return await invokeKubectlCommand(this.kubectl, this.kubeConfigFilePath, cpcommand);
+        });
+
+        if (failed(nodeExplorerResult)) {
+            vscode.window.showErrorMessage(`Failed to apply copy command: ${nodeExplorerResult.error}`);
+            return;
+        }
+
+        const goToFolder = "Go to Folder";
+        vscode.window.showInformationMessage(`Successfully copied the Retina Capture data to ${localCpPath}`, goToFolder)
+            .then(selection => {
+                if (selection === goToFolder) {
+                    open(localCpPath);
+                    // vscode.commands.executeCommand('vscode.openFolder', vscode.Uri.file(`${localCpPath}`));
```

Comment:
we can remove this commented line once open works for all Operation System's like windows, mac and Linux. Leaving this comment as placeholder.